jsapp/js/actions.es6

```diff
@@ -175,6 +181,73 @@ actions.resources.createImport.listen((params, onCompleted, onFailed) => {
     });
 });
 
+/*
+ * Form media endpoint actions
+ */
+actions.media.uploadMedia.listen((uid, formMediaJSON, callbacks = {}) => {
+  dataInterface.postFormMedia(uid, formMediaJSON)
+    .done(() => {
+      if (callbacks.onComplete) {
```

Comment:
After all the comments below are handled please check if any of these `onComplete` and `onFail` callbacks in the media actions are being used - if not, please make sure to remove them.